.changeset/loud-jars-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+"@builder.io/sdk-angular": patch
+---
+
+Resolved assertion error encountered when dynamically switching components
```

Comment:
This changeset is too low-level. I prefer changesets to focus on what customer-facing impact they have. In this case, the fact that Visual Editing could cause a crash:

```suggestion
Fix: crashes when visually editing blocks (encountered when SDK dynamically switched HTML elements)
```